plugins/@grouparoo/mongo/public/templates/query-source/{{{id}}}.js.template

```diff
@@ -0,0 +1,42 @@
+exports.default = async function buildConfig() {
+  return [
+    /**
+     * --------------------
+     * Source
+     * --------------------
+     */
+    {
+      class: "source",
+      id: {{{id}}},
+      name: {{{id}}},
+      type: "{{{__pluginName}}}-query-import",
+      appId: {{{appId}}}, // The ID of the App this Source uses - e.g. `appId: "data_warehouse"`
+
+      options: {
+        table: "..." // The Mongo collection in use.
+      }
+    },
+
+    /**
+     * ---------------
+     * Schedule
+     * ---------------
+     *
+     * A Schedule tells Grouparoo how frequently to check the Source for updated
+     * data and import it into the application database. If you would like to
+     * set a Schedule for this Source, uncomment the following section.
+     */
+    // {
+    //   id: {{{schedule_id}}},
+    //   name: {{{schedule_id}}},
+    //   class: "schedule",
+    //   sourceId: {{{id}}}, // The ID of the Source above
+    //   recurring: true, // should this Schedule regularly run?
+    //   recurringFrequency: 1000 * 60 * 15, // 15 minutes, in ms
+    //   options: {
+    //     query: "SELECT id FROM users WHERE updated_at >= (NOW() - INTERVAL '2 day')", // A SQL query to return that includes only the IDs of properties to check.
```

Comment:
@pauloouriques do you have an example query we might use in this template for a mongo query source?